PromisesObjC.podspec

```diff
@@ -28,10 +28,12 @@ Pod::Spec.new do |s|
   }
 
   s.test_spec 'Tests' do |ts|
+    ts.platforms = {:ios => '8.0', :osx => '10.10', :tvos => '9.0'}
```

Comment:
Is it possible to leave out the version specifiers? This would reduce maintenance when bumping versions.

Reply:
Unfortunately it does not seem that we can leave out version specifiers in test_spec... I tried it.

Reply:
Hey Koichi, does this work?

```
ts.platforms = {:ios => nil, :osx => nil, :tvos => nil}
```

(from https://github.com/CocoaPods/CocoaPods/issues/4185)

Reply:
It works as expected. Thanks for your input! 
Can you see the updated PR?